app/src/main/java/ch/epfl/qedit/view/edit/EditMapsActivity.java

```diff
@@ -0,0 +1,89 @@
+package ch.epfl.qedit.view.edit;
+
+import android.os.Bundle;
+import android.view.Menu;
+import androidx.appcompat.app.AppCompatActivity;
+import ch.epfl.qedit.R;
+import com.google.android.gms.maps.CameraUpdate;
+import com.google.android.gms.maps.CameraUpdateFactory;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.MarkerOptions;
+
+public class EditMapsActivity extends AppCompatActivity implements OnMapReadyCallback {
+    public static final String LATITUDE = "ch.epfl.qedit.view.edit.LATITUDE";
+    public static final String LONGITUDE = "ch.epfl.qedit.view.edit.LONGITUDE";
+
+    private GoogleMap mMap;
```

Comment:
Is there a particular reason for the 'm' in from of the variable name ? Otherwise, I think we can make it more consistent with the rest of the application and remove the 'm'. (Yes, I am very picky with the syntax ;-), I don't know why myself.)

Reply:
No it was generated automatically by android